### ISSUE TITLE
Conda env create on MacOS 12.1 (Apple M1/ARM) fails with unresolved packages

### ISSUE THREAD

**Issue by CharlieMcVicker** [CherokeeLanguage/IMS-Toucan#4]
Trying to get my development environment set up and I can't seem to get conda to find these packages. Need to run to an appt right now, but I thought I'd make the issue I'm having known!

Not surprised if this is arm related.

```
IMS-Toucan % conda env create -f environment.yml
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - mkl_random=1.2.2
  - intel-openmp=2021.4.0
  - libstdcxx-ng=9.3.0
  - mkl=2021.4.0
  - libgcc-ng=9.3.0
  - mkl_fft=1.3.1
  - mkl-service=2.4.0
  - libgomp=9.3.0
  - _libgcc_mutex=0.1
  - cudatoolkit=11.3.1
  - torchaudio=0.10.1
  - _openmp_mutex=4.5
 ```
Any help appreciated!

ᏩᏙ

**Comment by michael-conrad** [comment]
yeah... based on my googling for "apple m1 cudatoolkit" you are probbly in a show stopping situation.

**Comment by michael-conrad** [comment]
My first suggestion would be to remove the package version numbers on the offending packages in the environment.yml file. and also completely remove the cudatoolkit package.

**Comment by CharlieMcVicker** [comment]
Hmm. Yeah, I agree. If you'd like, I could take a look at containerizing this while I'm at it. I already have to check the dependenceis for another OS, might as well be Linux and then it'll run anywhere, including in the cloud if you ever want that. I know you've mentioned performance being a constraint, though. Still, I'm sort of interested to see what happens and what that performance hit is like.

**Comment by michael-conrad** [comment]
My biggest concern for you is your ability to generate needed audio for your web app projects.
